src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,38 @@
+.page {
+  margin: 0;
+  padding: 0;
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: $font-size;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  width: $calendar-width;
+  padding: $calendar-padding;
+  gap: $gap;
```

Comment:
The variable `$gap` is used but not defined in this file. Ensure it is defined or imported from another SCSS file.